hibernate-core/src/test/java/org/hibernate/test/legacy/ParentChildTest.java

```diff
@@ -1095,6 +1090,7 @@ public void testDeleteEmpty() throws Exception {
 	}
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done